app.rb

```diff
@@ -0,0 +1,14 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "pg" # postrgres db library
+require "active_record" # the ORM
+require "pry" # for debugging
+
+require_relative "db/connection" # require the db connection file that connects us to PSQL, prior to loading models
+require_relative "models/apartment" # require the Student class definition that we defined in the models/student.rb file
+require_relative "models/tenant" # require the Student class definition that we defined in the models/student.rb file
+
+
+# This will put us into a state of the pry REPL, in which we've established a connection
+# with the wdi database and have defined the Student Class as an
+# ActiveRecord::Base class.
+binding.prys
```

Comment:
Does this work with a plural `prys` at the end? Even if it does, you should use the singular `binding.pry`